### DISCUSSION TITLE
configurable button

### DISCUSSION THREAD

**Discussion by Vince56190** [agronick/aa-torque#27] [Ideas]
Possibility of adding a configurable button (we set the PID ourselves) to activate specific functions: sport mode, heated steering wheel

**Comment by agronick** [comment]
You can enable your heated steering wheel with a PID?

**Comment by Vince56190** [comment]
Why not , all my car control  are on the OEM touch screen. I Can start the car , Close Windows with an Android App so  i just have to find the PID for sport mode or other fonction liké heating steering wheel, heating Seat... . Many app like AlfaOdb on Android could activate or test  fonction via  ODB plug , I think  they use PID to do that.Le 7 oct. 2023 17:34, Kyle Agronick ***@***.***> a écrit :
You can enable your heated steering wheel with a PID?

**Comment by agronick** [comment]
I don't think they are PIDs. I think PIDs just get metrics. You can inject custom ELM327 commands in Torque but it is very advanced. You would need to sniff the packets.

**Comment by juliaabrams** [comment]
Which continents are we talking about exactly?

**Comment by iandavo2** [comment]
Not sure about what the OP wants to do but I'd love to see buttons to do the same as can be configured in Torque eg Reset Fuel, Reset Trip

**Reply by agronick** [reply]
I don't see a way to send reset commands through their service. https://github.com/mpicco/torquescan/blob/master/app/src/main/aidl/org/prowl/torque/remote/ITorqueService.aidl I'm going to look into adding buttons that can run arbitrary commands but it will be pretty advanced.
